build_tools/benchmarks/reporting/parse_shark_benchmarks.py

```diff
@@ -0,0 +1,305 @@
+  verison_html = verison_html + '<i>SHARK version: ' + args.shark_version + '</i><br/>'
+  verison_html = verison_html + '<i>last updated: ' + datetime.today().strftime(
+      '%Y-%m-%d') + '</i><br/><br/>'
+  html = generate_header_and_legend(verison_html)
+
+  # Generate Server CPU Static.
+  html = html + generate_table(
+      args.cpu_iree_csv,
+      args.cpu_baseline_csv,
+      shark_csv=args.cpu_shark_csv,
+      shape_type='static',
+      device='cpu',
+      title='Server Intel Ice Lake CPU (Static Shapes)')
+
+  # Generate Server GPU Static.
+  html = html + generate_table(
+      args.gpu_iree_csv,
+      args.gpu_baseline_csv,
+      shark_csv=args.gpu_shark_csv,
+      shape_type='static',
+      device='cuda',
+      title='Server NVIDIA Tesla A100 GPU (Static Shapes)')
+
+  # Generate Server CPU Dynamic.
+  html = html + generate_table(
+      args.cpu_iree_csv,
+      args.cpu_baseline_csv,
+      shark_csv=args.cpu_shark_csv,
+      shape_type='dynamic',
+      device='cpu',
+      title='Server Intel Ice Lake CPU (Dynamic Shapes)')
+
+  # Generate Server GPU Dynamic.
+  html = html + generate_table(
+      args.gpu_iree_csv,
+      args.gpu_baseline_csv,
+      shark_csv=args.gpu_shark_csv,
+      shape_type='dynamic',
+      device='cuda',
+      title='Server NVIDIA Tesla A100 GPU (Dynamic Shapes)')
+
+  f = open(args.output_path, 'w')
+  f.write(html)
+  f.close()
+
+
+def parse_args():
+  parser = argparse.ArgumentParser()
+  parser.add_argument(
+      '--cpu_shark_csv',
+      type=str,
+      default=None,
+      help=
+      'The path to the csv file with CPU benchmarking results from the SHARK runtime.'
+  )
+  parser.add_argument(
+      '--cpu_iree_csv',
+      type=str,
+      default=None,
+      help='The path to the csv file with CPU benchmarking results from IREE.')
+  parser.add_argument(
+      '--cpu_baseline_csv',
+      type=str,
+      default='data/icelake_baseline_0919.csv',
+      help='The path to the csv file containing baseline CPU results.')
+  parser.add_argument(
+      '--gpu_shark_csv',
+      type=str,
+      default=None,
+      help=
+      'The path to the csv file with GPU benchmarking results from the SHARK runtime.'
+  )
+  parser.add_argument(
+      '--gpu_iree_csv',
+      type=str,
+      default=None,
+      help='The path to the csv file with CPU benchmarking results from IREE.')
+  parser.add_argument(
+      '--gpu_baseline_csv',
+      type=str,
+      default='data/a100_baseline_0919.csv',
+      help='The path to the csv file containing baseline GPU results.')
+  parser.add_argument('--iree_version',
+                      type=str,
+                      default=None,
+                      help='The IREE version.')
+  parser.add_argument('--shark_version',
+                      type=str,
+                      default=None,
+                      help='The SHARK version.')
+  parser.add_argument(
+      '--output_path',
+      type=str,
```

Comment:
```suggestion
      type=pathlib.Path,
```

if using pathlib, this just works:tm: